pkg/config/init_config_wasm.go

```diff
@@ -0,0 +1,21 @@
+//go:build js && wasm
+// +build js,wasm
+
+package config
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/spf13/viper"
```

Comment:
Please keep this package in the second import group.

Reply:
Fixed.